<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeddeiNiteBase</class>
 <widget class="QMainWindow" name="GeddeiNiteBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Geddei Nite</string>
  </property>
  <widget class="QWidget" name="widget">
   <layout class="QGridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="ProcessorsView" name="theView">
      <property name="renderHints">
       <set>QPainter::Antialiasing|QPainter::TextAntialiasing</set>
      </property>
      <property name="dragMode">
       <enum>QGraphicsView::ScrollHandDrag</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="MenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>644</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="fileNew"/>
    <addaction name="fileOpen"/>
    <addaction name="fileSave"/>
    <addaction name="fileSaveAs"/>
    <addaction name="separator"/>
    <addaction name="filePrint"/>
    <addaction name="separator"/>
    <addaction name="fileExit"/>
   </widget>
   <widget class="QMenu" name="editMenu">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="editUndo"/>
    <addaction name="editRedo"/>
    <addaction name="separator"/>
    <addaction name="editCut"/>
    <addaction name="editCopy"/>
    <addaction name="editPaste"/>
    <addaction name="editRemove"/>
    <addaction name="separator"/>
    <addaction name="editFind"/>
    <addaction name="separator"/>
    <addaction name="editPreferences"/>
   </widget>
   <widget class="QMenu" name="modeMenu">
    <property name="title">
     <string>&amp;Mode</string>
    </property>
    <addaction name="modeStop"/>
    <addaction name="modeRun"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="toolsMenu">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="toolsDeployPlayer"/>
   </widget>
   <widget class="QMenu" name="viewMenu">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="viewDynamicDisplay"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="editMenu"/>
   <addaction name="viewMenu"/>
   <addaction name="toolsMenu"/>
   <addaction name="modeMenu"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="fileNew"/>
   <addaction name="fileOpen"/>
   <addaction name="fileSave"/>
   <addaction name="separator"/>
   <addaction name="modeStop"/>
   <addaction name="modeRun"/>
  </widget>
  <widget class="QDockWidget" name="theDockProperties">
   <property name="windowTitle">
    <string>Properties</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="theType">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="theName"/>
     </item>
     <item>
      <widget class="QTableWidget" name="theProperties">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="gridStyle">
        <enum>Qt::DotLine</enum>
       </property>
       <property name="cornerButtonEnabled">
        <bool>false</bool>
       </property>
       <property name="rowCount">
        <number>2</number>
       </property>
       <property name="columnCount">
        <number>1</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>100</number>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>100</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>19</number>
       </attribute>
       <attribute name="verticalHeaderMinimumSectionSize">
        <number>19</number>
       </attribute>
       <row>
        <property name="text">
         <string>New Row</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>New Row</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>New Column</string>
        </property>
       </column>
       <item row="0" column="0">
        <property name="text">
         <string>dwqdwqdwqdwqdqdqdq</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="theDockProcessors">
   <property name="windowTitle">
    <string>Processors</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="ProcessorView" name="theProcessors">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::DragOnly</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="uniformItemSizes">
        <bool>true</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="fileNew">
   <property name="icon">
    <iconset>
     <normaloff>image1</normaloff>image1</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="iconText">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="name" stdset="0">
    <cstring>fileNewAction</cstring>
   </property>
  </action>
  <action name="fileOpen">
   <property name="icon">
    <iconset>
     <normaloff>image2</normaloff>image2</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="iconText">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="name" stdset="0">
    <cstring>fileOpenAction</cstring>
   </property>
  </action>
  <action name="fileSave">
   <property name="icon">
    <iconset>
     <normaloff>image3</normaloff>image3</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="iconText">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="name" stdset="0">
    <cstring>fileSaveAction</cstring>
   </property>
  </action>
  <action name="fileSaveAs">
   <property name="icon">
    <iconset>
     <normaloff>image4</normaloff>image4</iconset>
   </property>
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
   <property name="iconText">
    <string>Save &amp;As...</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
   <property name="name" stdset="0">
    <cstring>fileSaveAsAction</cstring>
   </property>
  </action>
  <action name="filePrint">
   <property name="icon">
    <iconset>
     <normaloff>image5</normaloff>image5</iconset>
   </property>
   <property name="text">
    <string>&amp;Print...</string>
   </property>
   <property name="iconText">
    <string>Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
   <property name="name" stdset="0">
    <cstring>filePrintAction</cstring>
   </property>
  </action>
  <action name="fileExit">
   <property name="icon">
    <iconset>
     <normaloff>image6</normaloff>image6</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="iconText">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="name" stdset="0">
    <cstring>fileExitAction</cstring>
   </property>
  </action>
  <action name="editUndo">
   <property name="icon">
    <iconset>
     <normaloff>image7</normaloff>image7</iconset>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="iconText">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
   <property name="name" stdset="0">
    <cstring>editUndoAction</cstring>
   </property>
  </action>
  <action name="editRedo">
   <property name="icon">
    <iconset>
     <normaloff>image8</normaloff>image8</iconset>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="iconText">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
   <property name="name" stdset="0">
    <cstring>editRedoAction</cstring>
   </property>
  </action>
  <action name="editCut">
   <property name="icon">
    <iconset>
     <normaloff>image9</normaloff>image9</iconset>
   </property>
   <property name="text">
    <string>&amp;Cut</string>
   </property>
   <property name="iconText">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
   <property name="name" stdset="0">
    <cstring>editCutAction</cstring>
   </property>
  </action>
  <action name="editCopy">
   <property name="icon">
    <iconset>
     <normaloff>image10</normaloff>image10</iconset>
   </property>
   <property name="text">
    <string>C&amp;opy</string>
   </property>
   <property name="iconText">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
   <property name="name" stdset="0">
    <cstring>editCopyAction</cstring>
   </property>
  </action>
  <action name="editPaste">
   <property name="icon">
    <iconset>
     <normaloff>image11</normaloff>image11</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="iconText">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
   <property name="name" stdset="0">
    <cstring>editPasteAction</cstring>
   </property>
  </action>
  <action name="editFind">
   <property name="icon">
    <iconset>
     <normaloff>image12</normaloff>image12</iconset>
   </property>
   <property name="text">
    <string>&amp;Find...</string>
   </property>
   <property name="iconText">
    <string>Find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
   <property name="name" stdset="0">
    <cstring>editFindAction</cstring>
   </property>
  </action>
  <action name="viewDynamicDisplay">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>image17</normaloff>image17</iconset>
   </property>
   <property name="text">
    <string>&amp;Dynamic display</string>
   </property>
   <property name="iconText">
    <string>Snap To Grid</string>
   </property>
   <property name="name" stdset="0">
    <cstring>viewDynamicDisplayAction</cstring>
   </property>
  </action>
  <action name="editRemove">
   <property name="icon">
    <iconset>
     <normaloff>image18</normaloff>image18</iconset>
   </property>
   <property name="text">
    <string>R&amp;emove</string>
   </property>
   <property name="iconText">
    <string>Remove</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
   <property name="name" stdset="0">
    <cstring>editRemoveAction</cstring>
   </property>
  </action>
  <action name="toolsDeployPlayer">
   <property name="icon">
    <iconset>
     <normaloff>image19</normaloff>image19</iconset>
   </property>
   <property name="text">
    <string>&amp;Deply File Player</string>
   </property>
   <property name="iconText">
    <string>Deply File Player</string>
   </property>
   <property name="name" stdset="0">
    <cstring></cstring>
   </property>
  </action>
  <action name="editPreferences">
   <property name="icon">
    <iconset>
     <normaloff>image21</normaloff>image21</iconset>
   </property>
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
   <property name="iconText">
    <string>Preferences</string>
   </property>
   <property name="name" stdset="0">
    <cstring>editPreferencesAction</cstring>
   </property>
  </action>
  <actiongroup name="modeActionGroup">
   <action name="modeStop">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>image13</normaloff>image13</iconset>
    </property>
    <property name="text">
     <string>&amp;Edit</string>
    </property>
    <property name="iconText">
     <string>Edit</string>
    </property>
    <property name="shortcut">
     <string>Esc</string>
    </property>
    <property name="name" stdset="0">
     <cstring></cstring>
    </property>
   </action>
   <action name="modeRun">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>image15</normaloff>image15</iconset>
    </property>
    <property name="text">
     <string>&amp;Run</string>
    </property>
    <property name="iconText">
     <string>Run</string>
    </property>
    <property name="shortcut">
     <string>F5</string>
    </property>
    <property name="name" stdset="0">
     <cstring>modeRunAction</cstring>
    </property>
   </action>
   <property name="name" stdset="0">
    <cstring>modeActionGroup</cstring>
   </property>
  </actiongroup>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ProcessorView</class>
   <extends>QListWidget</extends>
   <header>processorview.h</header>
  </customwidget>
  <customwidget>
   <class>ProcessorsView</class>
   <extends>QGraphicsView</extends>
   <header>processorsview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
